cloud/deploys/dev_azure/README.md

```diff
@@ -3,38 +3,59 @@ This is a deploy directory that allows individuals to create a deploy of the
 to allow for everyone to have their own app. 
 
-The process to deploy is similar
-to the staging process but we do not have to do the shared state set up. It 
-should be `terraform init` and then `terraform apply`
+# Running the dev deploy
+## Copy the civiform_config.example.sh 
+Copy the civiform_config.example.sh into civiform_config.sh and
+ change the required variables. 
 
-# Local Docker Build to Remote Azure Deploy
-If you want to do local onto terraform we build/tag/deploy the docker image 
-and then update the azure app service to point to the local image. 
+## Get access to correct technologies
+You will need to reach out to team members to get accounts for the following:
+- Azure
+- AWS
+- Login Radius Civiform-Staging 
 
-## 1. Build, Tag and Push the Docker Image
-This should take like 30 minutes (the push takes the longest).
+## Setup Login Radius For Local Development
+Go to [Login Radius Dashboard](https://dashboard.loginradius.com/) and click
+configure a civiform integration. Choose the outbound SSO Saml.
+
+From there add an app with Sp initiated login and pick a name (this gets put
+into the config as `LOGIN_RADIUS_SAML_APP_NAME`).
 
+To generate the private key for the form run, cat the file and put into dashboard.
 ```
-docker build -f prod.Dockerfile -t <IMAGE_TAG> --cache-from docker.io/civiform/civiform-browser-test:latest --build-arg BUILDKIT_INLINE_CACHE=1 .
-docker tag <IMAGE_TAG> <DOCKER_USERNAME>/<DOCKER_REPO_NAME>:<IMAGE_TAG>
-docker push <DOCKER_USERNAME>/<DOCKER_REPO_NAME>:<IMAGE_TAG>
+openssl genrsa -out private.key 2048
 ```
 
-## 2. Update the image name/tag for your remote azure deploy
-You can do this one of two ways. Terraform deploy or update via the azure 
-portal 
+For generating the cert run, cat the file and put into dashboard.
+```
+openssl req -new -x509 -key private.key -out certificate.cert -days 365 -subj /CN=civiform-staging.hub.loginradius.com
+```
 
-### Update via the .tfvars File and deploy
-After that you can change your .tfvars file to point to the docker tag you set
+We need to copy the details from a previous working setup in login radius 
+once we set up the certs so look back at the staging one to fill out.
+
+## Run
+After that you can start the setup by running and following the instructions:
 
 ```
-docker_repository_name = <DOCKER_REPO_NAME>
-docker_username        = <DOCKER_USERNAME>
+cloud/deploys/dev_azure/bin/setup  
 ```
 
-deploy via `terraform apply`
+# Local Docker Build to Remote Azure Deploy
+If you want to do local onto terraform we build/tag/deploy the docker image 
+and then update the azure app service to point to the local image. 
+
+## 1. Build, Tag and Push the Docker Image
+Run the following script which takes the image tag from your civiform_config.sh
+and builds, tags and pushes it up to docker hub. You will need a custom 
+docker hub in order to do this. Check with team on how to pay for docker hub pro.
```

Comment:
suggestion (blocking): also mention they need to set the `DOCKER_REPOSITORY` and `DOCKER_USERNAME` variables accordingly